README.md

```diff
@@ -29,15 +29,20 @@ Why?
 
 ## Usage
 
-To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
+### Dependencies
 
-    $ sudo gem install jekyll bundler
+Install [asdf-vm](https://asdf-vm.github.io/asdf/#/core-manage-asdf-vm) for your OS and now using asdf-vm, install [ruby](https://github.com/asdf-vm/asdf-ruby).
+
+### Cloning and running
+
+To clone the repository and run this website on your local machine, type the following commands:
+    
+    $ gem install jekyll bundler
```

Comment:
Move this somewhere before cloning the repository and after the dependency section, ideally it should be in the dependency section.

Reply:
Can you plz tell how can i do that like should i make point no. 3 and just write the command ?

Reply:
Split dependencies section in points, this will create 3 points in the section.
1. for installing asdf-vm
2. for installing ruby
3. for jekyll and bundler